.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
recommend to add vscode settings and workspace to gitignore

Reply:
done 👍